games/pong/src/main.zig

```diff
@@ -16,10 +25,27 @@ pub fn main() anyerror!void {
 
     rl.setTargetFPS(60); // Set our game to run at 60 frames-per-second
     //--------------------------------------------------------------------------------------
+    var gpa: std.heap.GeneralPurposeAllocator(.{}) = .init;
```

Comment:
Is this the correct way to initialise the GPA? 
Also, is the GPA the appropriate one here?